### DISCUSSION TITLE
Winget.exe is missing from my Win10 and Win11 VM's

### DISCUSSION THREAD

**Discussion by Swe-it** [microsoft/winget-cli#2900] [Q&A] [answered]
I have started laborate with Winget and Intune, I know that I will need this `C:\Program Files\WindowsApps\Microsoft.DesktopAppInstaller_*_x64__8wekyb3d8bbwe\Winget.exe`  to make it run but when I set up my test machine both Win10 Pro and Win11 Pro Winget is missing for the System to run. And for the user on the VM as well it's missing.

I have looked on a couple of my users computers (Win10's) and they have it but to do testing I need it also on my VM's

Does anybody know how this can be?

**Comment by Masamune3210** [comment, marked as answer]
Make sure the App Installer package is up to date, Winget is included in the package

**Reply by Swe-it** [reply]
Thank you, I ran:

`Add-Appxpackage .\Microsoft.DesktopAppInstaller_8wekyb3d8bbwe.msixbundle`

And now it works :)

**Reply by Swe-it** [reply]
It doesn't seem to work on my Win10 VM, it worked on the Win11.

I had to first install `VC_redist.x64` then run the update but it didn't give me Winget.exe.

**Reply by Swe-it** [reply]
But when I run Winget Installation från Intune on my Win10 it works even though I can't see Winget.exe 

:)
